give-addon-boilerplate.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /**
  * Plugin Name: Give - Addon Boilerplate
- * Plugin URI:  https://github.com/impress-org/give-addon-boilerplate
+ * Plugin URI:  https://github.com/impress-org/givewp-addon-boilerplate
  * Description: A demo Addon to serve as a boilerplate for devs to better understand how to extend the Give Donation plugin for WordPress.
  * Version:     1.0
  * Author:      WordImpress, LLC
```

Comment:
The author should be Impress instead of WordImpress.

Reply:
fixed